src/wp-content/themes/twentytwentyfour/functions.php

```diff
@@ -18,7 +18,6 @@
 	 *
 	 * @return void
 	 * @since Twenty Twenty-Four 1.0
```

Comment:
nit: Just noticing this now, but @since should be before @return here and in the other spots. See: https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods

Reply:
Ah, never mind. I see that WordPress/twentytwentyfour#693 been opened. Approved and merged.

Reply:
Thank you!

Reply:
```
@return void
```

should also be removed. Will open an issue to fix in the TT4 repo.

Reply:
@hellofromtonya According to https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods `Note: @return void should not be used outside of the default bundled themes.`

Reply:
So I think it's ok